cpp/CMakeLists.txt

```
@@ -48,6 +48,7 @@ option(BUILD_TESTS "Configure CMake to build tests" ON)
 option(BUILD_BENCHMARKS "Configure CMake to build (google & nvbench) benchmarks" OFF)
 option(BUILD_SHARED_LIBS "Build cuDF shared libraries" ON)
 option(JITIFY_USE_CACHE "Use a file cache for JIT compiled kernels" ON)
+option(CUDF_BUILD_WHEELS "Whether we're building libcudf to go in a wheel for pypi" OFF)
```

Avoid "we". Also no need to specify PyPI here.
```suggestion
option(CUDF_BUILD_WHEELS "Whether libcudf is being built for a Python wheel" OFF)
```